axelor-base/src/main/java/com/axelor/apps/base/service/imports/ImportConfigurationCallableService.java

```diff
@@ -46,7 +48,9 @@ public ImportHistory call() throws Exception {
       return importHistory;
     } catch (Exception e) {
       onRunnerException(e);
-      throw e;
+      throw new AxelorException(
+          TraceBackRepository.CATEGORY_INCONSISTENCY,
+          I18n.get(BaseExceptionMessage.IMPORT_CONFIGURATION_WRONG_FILE_TYPE_MESSAGE));
```

Comment:
The catch block could have caught an exception for a different reason. I think we should do an explicit check of the file extension.